### ISSUE TITLE
FileNotFoundError in Dataloader

### ISSUE THREAD

**Issue by armanbabayan** [roboflow/notebooks#170]
### Search before asking

- [X] I have searched the Roboflow Notebooks [issues](https://github.com/roboflow-ai/notebooks/issues) and found no similar bug report.

### Notebook name

train_yolo_nas_on_custom_dataset.ipynb

### Bug

![Screenshot from 2023-07-18 12-33-38](https://github.com/roboflow/notebooks/assets/84587490/3a77dac9-bc24-49c5-ad2a-22a62d5abad1)
I am encountering a 'FileNotFoundError' in Dataloader while training my YOLO-NAS model on custom dataset. However, the file mentioned in the error message does exist in the directory. After training model for 13 epochs, I received the following error. I need assistance in resolving this issue.  

### Environment

Google Colab

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by SkalskiP** [comment]
Hi, @armanbabayan 👋🏻! 

Could you run this code:

```python
import os

file_path = "/content/gdrive/MyDrive/YOLO NAS/small data/train/images/4721 9d6e3525159ac42240be816022c6101534962.png"
file_exists = os.path.isfile(file_path)

print(file_exists)
```

**Comment by armanbabayan** [comment]
Hi, @SkalskiP !
Thank you for reply. Actually I run this code and it returns True:

import os

file_path = "/content/gdrive/MyDrive/YOLO_NAS/small_data/train/images/4721_9d6e3525159ac42240be816022cc6101534962.png"
file_exists = os.path.isfile(file_path)

print(file_exists)

**Comment by SkalskiP** [comment]
@armanbabayan, that's super weird. It doesn't relate to our notebook but to YOLO-NAS internals. Unfortunately, we won't be able to help you with that. Please create an issue here: https://github.com/Deci-AI/super-gradients. You can refer to this issue for more visibility.

**Comment by armanbabayan** [comment]
@SkalskiP Ok, thank you very much!

**Comment by SkalskiP** [comment]
@armanbabayan I'm sorry I couldn't be more helpful.
